plenum/server/node.py

```diff
@@ -2347,8 +2347,7 @@ def processPropagate(self, msg: Propagate, frm):
         logger.debug("{} received propagated request: {}".
                      format(self.name, msg))
 
-        reqDict = msg.request
-        request = self.client_request_class(**reqDict)
+        request = self.client_request_class(**msg.request)
```

Comment:
This should be moved to the some previous scheduler action before authentication to have a key (digest) that is stored in auth cache of ReqAuthenticator.